ibis/expr/api.py

```diff
@@ -168,6 +168,16 @@
 
 
 NA = null()
+"""
+Create a NULL/NA scalar.
```

Comment:
```suggestion
"""The NULL scalar.
```

I think this may mislead people into calling it if they don't read the example.